site/book/01-getting-started/02-quickstart.md

```diff
@@ -7,22 +7,40 @@ configuration using the underlying Git version control system.
 
 First, let's fetch the _kpt package_ from Git to your local filesystem:
 
+{{% hide %}}
+
+<!-- @makeWorkplace @verifyGuides-->
+```
+# Set up workspace for the test.
+TEST_HOME=$(mktemp -d)
+cd $TEST_HOME
+touch output.txt
+
+function expectedOutput() {
+  if [ "$(echo "$@")" == "$(cat output.txt)" ]; then echo 0; else echo 1; fi
+}
+```
+
+{{% /hide %}}
+
+<!-- @pkgGet @verifyGuides-->
 ```shell
-$ kpt pkg get https://github.com/GoogleContainerTools/kpt/package-examples/nginx@v0.4
-$ cd nginx
+kpt pkg get https://github.com/GoogleContainerTools/kpt/package-examples/nginx@v0.4
+cd nginx
 ```
 
 `kpt pkg` commands provide the functionality for working with packages on Git and on your local
 filesystem.
 
 Next, let's quickly view the content of the package:
 
+<!-- @pkgTree @verifyGuides-->
 ```shell
-$ kpt pkg tree
-PKG: nginx
+kpt pkg tree > output.txt
```

Comment:
This is visible to users in the docs which shouldn't be. Add extra section for each command in {{% hide %}} section to duplicate what command does and verify the output there.

Reply:
Sure, I have added extra sections as suggested.